pkg/aspect/root/flags/noable_bool.go

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2022 Aspect Build Systems, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package flags
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/pflag"
+)
+
+const (
+	BoolFlagTrue  = "true"
+	BoolFlagFalse = "false"
+	BoolFlagYes   = "yes"
+	BoolFlagNo    = "no"
+	BoolFlag1     = "1"
+	BoolFlag0     = "0"
+)
+
+func RegisterNoableBool(flags *pflag.FlagSet, name string, value bool, usage string) *bool {
+	return RegisterNoableBoolP(flags, name, "", value, usage)
+}
+
+// Register a boolean flag that supports Bazel option parsing.
```

Comment:
Nit: not a valid godoc comment.

Reply:
The automerge happened. However, I am willing to fix the issue. I don't see what the problem is with the comment. What would you recommend that I change?

Reply:
```// RegisterNoableBoolP registers a boolean flag ...```

I wonder why our current linters are not catching those. Maybe I missed some configuration.

Reply:
I also changed the merge rules to require that comments are resolved before auto-merge happens.